tsconfig.json

```diff
@@ -1,21 +1,23 @@
 {
   "compilerOptions": {
-    "target": "esnext",
-    "module": "esnext",
-    "moduleResolution": "node",
-    "strict": true,
-    "jsx": "preserve",
-    "sourceMap": true,
-    "resolveJsonModule": true,
-    "esModuleInterop": true,
-    "skipLibCheck": true,
     "allowJs": true,
+    "esModuleInterop": true,
+    "jsx": "preserve",
     "lib": ["esnext", "dom"],
-    "types": ["vite/client"],
+    "module": "esnext",
+    "moduleResolution": "node",
+    "noImplicitAny": true,
+    "noUnusedLocals": true,
```

Comment:
Mostly I'm just alphabetizing, but these `noImplicitAny` and `noUnuseLocals` properties are new.